.github/workflows/build_test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [8.x, 10.x, 12.x, 14.x, 16.x, 17.x]
```

Comment:
ESLint requires a minimum Node Version of 12.

All previous versions are beyond End Of Life and I don't think we need to continue to support them. In fact 12 EOL was over 1 month ago today. We can add additional modern versions in a separate ticket.

Reply:
This is a dependency for testing purpose. at the same time, I used `ls-engines` to check the dependencies graph and it shows `>= 6.14`.
Why would we change the compatibility right now (we can add `ls-engines` verification to the pipeline, etc) as the package did not change as a fact?

Reply:
I followed your suggestion and only removed the out of date node versions from the lint step.